pages/pipelines/build_exports.md

```diff
@@ -83,10 +83,33 @@ To configure build exports for your organization, you'll need to prepare an S3 b
 
 Your Buildkite Organization ID (UUID) can be found on the settings page described in the next section.
 
+### Prepare your Google Cloud Storage bucket
```

Comment:
@mitchbne exports was edited in its own wee PR :D
https://github.com/buildkite/docs/pull/2201/files

We can attempt to combine the two